src/daemon/dlt-daemon.c

```diff
@@ -1569,6 +1561,14 @@ int dlt_daemon_process_client_connect(DltDaemon *daemon,
         }
     }
 
+    snprintf(local_str,
+             DLT_DAEMON_TEXTBUFSIZE,
+             "New client connection #%d established, Total Clients : %d\n",
+             in_sock,
+             daemon_local->client_connections);
+    dlt_log(LOG_DEBUG, local_str);
```

Comment:
please use dlt_vlog instead of snprinft/dlt_log pair

Reply:
By this patch, I tried to output new log as both DLT message(with Apid: DLTD, Ctid: INTM) and internal log(by dlt_log()) with DEBUG log level. In this case, I cannot replace snprintf()/dlt_log() pair to dlt_vlog. How do you think? I thought these information should be visible as DLT message to know current status of dlt-daemon.

Reply:
Yes I saw the dlt_vlog/dlt_log implementation :-)
I guess you missed change of line 1570..
dlt_daemon_log_internal() can send DLT message from dlt-daemon but this function requires constructed string as the argument.

Reply:
ah yes, this is bad and should be changed later. than it is ok from my side.